hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -138,6 +140,14 @@ public static void checkSnapshotActive(SnapshotInfo snapInfo,
     }
   }
 
+  /**
+   * Set transactionInfo in snapshotInfo.
+   */
+  public static void setTransactionInfoInSnapshot(SnapshotInfo snapshot, TermIndex termIndex) throws IOException {
```

Comment:
nit: this is an unnecessary helper function to me. It could be a simple one-liner wherever needed.
```
snapshot.setLastTransactionInfo(TransactionInfo.valueOf(termIndex).toByteString());
```

Reply:
done